src/app/statement/statement.component.ts

```diff
@@ -0,0 +1,107 @@
+import { Component, ViewChild, AfterViewInit } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { CategoryRoutingName, CorePath } from '@app/shared/models';
+import { MatAccordion } from '@angular/material/expansion';
+
+type Statement = {
+  id: string;
+  title: string;
+  content: string;
+  isExpanded: boolean;
+};
+
+@Component({
+  selector: 'app-statement',
+  templateUrl: './statement.component.html',
+  styleUrls: ['./statement.component.scss'],
+})
+export class StatementComponent implements AfterViewInit {
+  categoryRoutingName = CategoryRoutingName;
+  corePath = CorePath;
+  accordionList: Array<Statement>;
+  shouldScroll: boolean;
+
+  @ViewChild('accordion') accordion!: MatAccordion;
+
+  constructor(private translateService: TranslateService) {
+    this.shouldScroll = false; // TODO: should be retrieved from the passed parameters
+    const shouldBeExpanded = !this.shouldScroll;
+
+    this.accordionList = [
```

Comment:
no business logic ina a constructor. it is enough to have it assign to component preperty.